docs/security.html

```diff
@@ -2089,6 +2089,144 @@ <h5 class="anchor-heading"><a id="operations_resources_and_protocols" class="anc
             <td>Topic</td>
             <td></td>
         </tr>
+        <tr>
+            <td>DESCRIBE_CLIENT_QUOTAS (48)</td>
+            <td>DescribeConfigs</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>ALTER_CLIENT_QUOTAS (49)</td>
+            <td>AlterConfigs</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>DESCRIBE_USER_SCRAM_CREDENTIALS (50)</td>
+            <td>Describe</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>ALTER_USER_SCRAM_CREDENTIALS (51)</td>
+            <td>Alter</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>VOTE (52)</td>
+            <td>ClusterAction</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>BEGIN_QUORUM_EPOCH (53)</td>
+            <td>ClusterAction</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>END_QUORUM_EPOCH (54)</td>
+            <td>ClusterAction</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>DESCRIBE_QUORUM (55)</td>
+            <td>ClusterAction</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>DESCRIBE_QUORUM (55)</td>
+            <td>Alter</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>ALTER_PARTITION (56)</td>
+            <td>ClusterAction</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>UPDATE_FEATURES (57)</td>
+            <td>ClusterAction</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>UPDATE_FEATURES (57)</td>
+            <td>Alter</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
```

Comment:
Duplicated entry.

Reply:
Thank you @showuon. Looking at the other ApiKeys, they are listed several times for each operation primitive and resource it checks, e.g FETCH. I followed the same format for DESCRIBE_QUORUM and UPDATE_FEATURES as they both had clusterAction set to true here (https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/common/protocol/ApiKeys.java#L101).  However now looking at them again, I'm not sure where ClusterAction operation is checked for them. I only see Describe or Alter operations being checked on Cluster resource.

Reply:
I think we should remove them since the permission is not required for this API key.